Kiosk/App/AppDelegate.swift

```diff
@@ -16,6 +16,10 @@ public class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(application: UIApplication!, didFinishLaunchingWithOptions launchOptions: NSDictionary!) -> Bool {
 
+        // Disable sleep timer
+        UIApplication.sharedApplication().idleTimerDisabled = true
```

Comment:
@sarahscott This is the only code change to my PR. 

I guess I could have used the `application` variable instead of the `sharedApplication()` singleton.. oh well.